libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -286,6 +286,8 @@ bool ASTJsonExporter::visit(ContractDefinition const& _node)
 		make_pair("abstract", _node.abstract()),
 		make_pair("baseContracts", toJson(_node.baseContracts())),
 		make_pair("contractDependencies", getContainerIds(_node.annotation().contractDependencies | ranges::views::keys)),
+		// Do not require call graph because the AST is also created for incorrect sources.
+		make_pair("emittedEvents", getContainerIds(_node.interfaceEvents(false))),
```

Comment:
I think this name is too misleading. It implies that only emitted events are included. Going back to this PR after a long time I actually got bitten by it myself. Initially I thought that it was the case and only realized that I'm wrong after reviewing tests.